thoth/storages/graph/postgres.py

```diff
@@ -3475,6 +3475,24 @@ def update_kebechet_github_installations_on_is_active(self, slug: str) -> bool:
                 return True
             return False
 
+    def get_active_kebechet_github_installations_repos(self) -> Dict[str, List[str]]:
+        """Get all active repositories names with active Kebechet installation.
+        
+         Examples:
+        >>> from thoth.storages import GraphDatabase
+        >>> graph = GraphDatabase()
+        >>> graph.get_active_kebechet_github_installations_repos()
+        ['repository_foo_fullname', 'repository_bar_fullname', ...]
+        """
+        with self._session_scope() as session:
+            active_installations = (
```

Comment:
@xtuchyna you can also use directly `with_entities` to receive directly the specific parameter.
```
active_installations_repos = (
     session.query(KebechetGithubAppInstallations)
     .filter(KebechetGithubAppInstallations.is_active.is_(True))
     .with_entities(KebechetGithubAppInstallations.slug)
     .all()
)

```

Reply:
This is better to do on the database side to avoid unnecessary serialization, rather than in python code.